client/js/socket-events/init.js

```diff
@@ -64,7 +64,8 @@ function openCorrectChannel(clientActive, serverActive) {
 
 	// Open window provided in location.hash
 	if (target.length === 0 && window.location.hash) {
-		target = $("#footer, #sidebar").find(`.chan[data-target="${escape(window.location.hash)}"]`);
+		target = $("#footer, #sidebar, #help")
```

Comment:
Would this break anything if you just searched on the entire body?

Reply:
I'm not... sure...

_Probably_ not, though [this `data-target` is suspicious](https://github.com/thelounge/lounge/blob/astorije/fix-window-on-load/client/views/chat.tpl#L2).

Reply:
That looks like something that can be removed. Doesn't it point to itself?